app/models/ability.rb

```diff
@@ -11,8 +11,11 @@ def initialize(user)
     can :view, Lecture
     can :view, Assignment
```

Comment:
Add a can :view, Semester because everyone should have the ability to see it, just not create update or destroy